applications/GeoMechanicsApplication/custom_elements/Policies.md

```diff
@@ -0,0 +1,137 @@
+# Policies
+
+## Introduction
+
+In our element structure, the strategy-pattern (also known as policy-pattern), is used to gain control and
```

Comment:
Perhaps we can refer to the Gang of Four book: "Design Patterns: Elements of Reusable Object-Oriented Software" by Erich Gamma, Richard Helm, Ralph Johnson and John Vlissides.

Reply:
Good suggestion, done!